File: C++/Daily Flash/week1/Day2_Jan14/code_c++/program3.cpp
/*
Program 3: Write a Program to Find whether the character is an alphabet, a
digit or a special character
Input: *
Output: * is a Special Character.
*/



#include <iostream>

int main() {
	char ch;
	std::cout << "Enter character: ";
	std::cin >> ch;
	
	if ((ch >=32 && ch <= 47) || (ch >=58 && ch <= 64 ) || (ch >= 91 && ch <= 96) || (ch >=123 && ch <=126))
		std::cout << ch <<" is a special character" << std::endl;
	else
		std::cout << ch <<" is not a special character" << std::endl;
	return 0;	
}

File: PYTHON/Daily Flash/week1/Day2_Jan14/code_python/program3.py
'''
Program 3: Write a Program to Find whether the character is an alphabet, a
digit or a special character
Input: *
Output: * is a Special Character.
'''

char = input("Enter character: ")
#print(int(char))

#if(char >= str(97)):
#   print("hello")


if( (char >= 'a' and char <= 'z') or (char >='A' and char <='Z' ) ):
        print(char ," is not a Special character")
elif (char >= '0' and char <= '9'):
        print(char ," is not a Special character")
else:
        print(char ," is a Special character")
